docker-compose.prod.yml

```
@@ -24,13 +24,14 @@ services:
       - seeds.env
 
   seeds_view:
-    build: ./view
+    image: node:20.11.0
+    working_dir: /app
```

のせニキが行なってくれた変更は変えなくても動いているので、戻しておいて下さい！
```suggestion
    working_dir: /app
```